app/views/templates/_form.html.erb

```diff
@@ -24,7 +24,7 @@
       <%= text_f f, :name, :disabled => @template.locked? %>
 
       <% if show_default? %>
-        <%= checkbox_f f, :default, :label=>_('Default'), :help_block => default_template_description %>
+        <%= checkbox_f f, :default, :label=>_('Default'), :help_inline => default_template_description %>
```

Comment:
No, I think I wanted to make it one of these little :grey_question: buttons, but it doesn't look like I did. Revert it if you want.